packages/zoe/src/contracts/multipoolAutoswap.js

```diff
@@ -23,611 +30,624 @@ import {
 // would first use the pool (X, C) then the pool (Y, C). There are no
 // liquidity pools between two secondary tokens.
 
-export const makeContract = harden(zcf => {
-  // This contract must have a "central token" issuer in the terms.
-  const CENTRAL_TOKEN = 'CentralToken';
+export const makeContract = harden(
+  /** @param {ContractFacet} zcf */ zcf => {
```

Comment:
This comment and the changes above are the only non-white spaces changes in the file. The rest are driven by eslint. This declaration allows vscode to typecheck arguments to all calls on zcf functions.